localstack/services/cloudformation/models/route53.py

```diff
@@ -38,9 +38,9 @@ def param_change_batch(params, **kwargs):
                 "HealthCheckId",
             ]
             attrs = select_attributes(params, attr_names)
-
-            if isinstance(attrs["TTL"], str):
-                attrs["TTL"] = int(attrs["TTL"])
+            if "TTL" in attrs:
+                if isinstance(attrs["TTL"], str):
+                    attrs["TTL"] = int(attrs["TTL"])
```

Comment:
```suggestion
            if isinstance(ttl := attrs.get("TTL"), str):
                attrs["TTL"] = int(ttl)
```
We could avoid the nested ifs with something like this.